samples/sensor/soc_voltage/README.rst

```diff
@@ -0,0 +1,30 @@
+.. _stm32_vref_sensor:
+
+SoC voltage sensor measurement
+##############################
+
+Overview
+********
+
+This sample reads one or more of the various voltage sensors an SoC might have and
+displays the results.
+
+Building and Running
+********************
+
+In order to run this sample, enable the sensor node that supports
+``SENSOR_CHAN_VOLTAGE`` and create an alias named ``volt-sensor0`` to link to the node.
+The tail ``0`` is the sensor number.  This sample support up to 16 sensors.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/sensor/stm32_vref_sensor
```

Comment:
Need to fix the app path

Reply:
Same as above, thanks for the in-depth look. :)